src/components/Frame/components/Toast/Toast.scss

```diff
@@ -17,6 +17,10 @@ $Backdrop-opacity: 0.88;
   }
 }
 
+.ActionLink {
+  margin-left: (spacing(extra-loose));
```

Comment:
```suggestion
  margin-left: spacing(extra-loose);
  margin-right: spacing();
```

I did notice that the spacing between the action and the close button is smaller than Sara's design. Throwing a right margin on there fits the design more closely. Also, I don't think those extra parentheses are needed.

![screen shot 2019-01-22 at 6 38 25 pm](https://user-images.githubusercontent.com/6239480/51579229-04ad8b80-1e75-11e9-9933-304e18da12ac.jpg)
![screen shot 2019-01-22 at 6 38 45 pm](https://user-images.githubusercontent.com/6239480/51579231-05462200-1e75-11e9-964c-37ed116b3b79.jpg)